devito/interfaces.py

```diff
@@ -18,8 +56,23 @@ def set_initializer(self, lambda_initializer):
         assert(callable(lambda_initializer))
         self.initializer = lambda_initializer
 
+    # function to generate a hased file name used for memmap file
+    def _generate_memmap_filenmae(self):
```

Comment:
Well, I guess then I will just use the name of the object then since the names are unique. If I mentioned because I remember seeing somewhere the an object of the same name were created twice, so I thought the name is not unique.